pandas/core/generic.py

```diff
@@ -1940,6 +1940,76 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None , replace=False, weights=None, seed=None):
+        """
+        Returns a random sample of rows from object. 
+        
+        Parameters
+        ----------
+        n: int, optional.
+            Number of rows to return. Cannot be used with frac.
+            Default = 5 if frac = None. 
+        frac: float. 
+            Share of rows to return. Cannot be used with n. 
+            Default = None.
```

Comment:
You don't need to explicitly give the default argument here -- it's already in the function signature.